test/build/base-url/base-url.test.js

```diff
@@ -27,18 +27,18 @@ describe('buildOptions.baseUrl', () => {
   it('import.meta.env works', () => {
     // env is present in index.js
     expect(files['/index.js']).toEqual(
-      expect.stringContaining(`import __SNOWPACK_ENV__ from './__snowpack__/env.js';`),
+      expect.stringContaining(`import * as __SNOWPACK_ENV__ from './__snowpack__/env.js';`),
```

Comment:
How did you feel about updating these tests? 👍🏻 or 👎🏻 ?

Reply:
easy! was going to say, definitely felt more intentional than just a big snapshot test